README.md

```diff
@@ -1,13 +1,21 @@
+# Biome - IntelliJ Plugin
 
-[Biome](https://biomejs.dev/) is a powerful tool designed to enhance your development experience. This plugin integrates seamlessly with many [JetBrains IDE's](#Supported IDEs) to provide some capabilities:
+[![IntelliJ IDEA Ultimate Version](https://img.shields.io/badge/IntelliJ%20IDEA%20Ultimate-2023.2.2-374151.svg?labelcolor=000&logo=intellij-idea&labelColor=black)](https://www.jetbrains.com/idea/)
+[![WebStorm Version](https://img.shields.io/badge/WebStorm-2023.2.2-1F3263.svg?labelcolor=000&logo=webstorm&labelColor=black)](https://www.jetbrains.com/webstorm/)
+[![AppCode Version](https://img.shields.io/badge/AppCode-2023.2.2-2380B0.svg?labelcolor=000&logo=appcode&labelColor=black)](https://www.jetbrains.com/objc/)
+[![PhpStorm Version](https://img.shields.io/badge/PhpStorm-2023.2.2-953D8C.svg?labelcolor=000&logo=phpstorm&labelColor=black)](https://www.jetbrains.com/phpstorm/)
+[![RubyMine Version](https://img.shields.io/badge/RubyMine-2023.2.2-A11523.svg?labelcolor=000&logo=ruby&labelColor=black)](https://www.jetbrains.com/ruby/)
 
--  See lints while you type
--  Apply code fixes (from mouse-over, <kbd title="Option">⌥</kbd>+<kbd  title="Enter">⏎</kbd> or <kbd title="Alt">Alt</kbd>+<kbd title="Enter">Enter</kbd>)
--  Reformat your code with <kbd>⌥⇧</kbd>+<kbd title="Cmd">⌘</kbd>+<kbd  title="L">L</kbd> or <kbd title="Ctrl">Ctrl</kbd>+<kbd title="Alt">Alt</kbd>+<kbd  title="L">L</kbd> (You can also format your [code on save](https://www.jetbrains.com/help/webstorm/reformat-and-rearrange-code.html#reformat-on-save))
+[Biome](https://biomejs.dev/) is a powerful tool designed to enhance your development experience. 
+This plugin integrates seamlessly with many [JetBrains IDE's](#supported-ides) to provide the following capabilities.
+
+- 💡 See lints while you type
+- 👨‍💻 Apply code fixes
+- 🚧 Reformat your code
 
 However, please note the following limitations:
 
-- Automatically applying code fixes on save
+- 💾 Automatically applying code fixes on save
```

Comment:
This will now be supported by the plugin, we can move it to the supported features list.

Can you also include "💾 Automatically format code on save"?

Reply:
Hi @victor-teles, I appreciate your suggestion. Feel free to suggest more changes with shortcuts or other new features . I've already implemented the automatic format code on save at feature list.